db/sqlinterfaces.c

```diff
@@ -6272,6 +6272,7 @@ int sql_check_errors(struct sqlclntstate *clnt, sqlite3 *sqldb,
     case SQLITE_COST_TOO_HIGH:
     case SQLITE_NO_TEMPTABLES:
     case SQLITE_NO_TABLESCANS:
+    case SQLITE_ANALYZE_ALREADY_RUNNING:
         *errstr = sqlite3_errmsg(sqldb);
```

Comment:
Is this error code handled in sqlite3_errmsg()?

Reply:
I believe so. I get this error message in this case which I added in a previous PR, and I can see sqlite3_errmsg returning this string: `[analyze t] failed with rc -22 Analyze could not run because it is already running on some table`

Reply:
https://github.com/bloomberg/comdb2/blob/535d3b466b5b911e1eb99dae33c278385a45f3b2/sqlite/src/comdb2build.c#L1601